### ISSUE TITLE
Auto-generated email

### ISSUE THREAD

**Issue by Souad314** [DataSeer/dataseer-web#294]
For when a standard_user Finishes a document, can there be an automated email triggered to let info@dataseer know that they've arrived at the Finish page?
Ideally the email would go to Sam or my DataSeer email depending on who is assigned, but if both of us work on the document, it may be confusing or not possible?

**Comment by NicolasKieffer** [comment]
Yes, automatic emails can be sent when a standard_user go on finish step.

Note: the email can be sent to one or more DS accounts if needed (e.g. all curators who worked on the document)

**Comment by timvines19** [comment]
Could you please have these emails go to info@dataseer.ai and to any other curators listed on the article?
